src/main/java/Request/RequestParser.java

```diff
@@ -0,0 +1,27 @@
+public class RequestParser {
+  private String method;
+  private RequestLine requestLine;
+  private String[] splitRequest;
+  private String uri;
+  private String version;
+
+  RequestParser(String requestString) {
+    this.splitRequest = requestString.split("\r\n");
+    parseRequestLine();
+  }
+
+  private void parseRequestLine() {
```

Comment:
In idiomatic Java, it's standard convention that private methods are declared below all public methods